### PR TITLE
Add instructions for AE table 3

### PR DESCRIPTION
**Pull request by tylergu**

### CODE REVIEW

File: AE.md [hunk 1]
```diff
@@ -47,9 +47,83 @@ cat zookeeper.json | grep "errors"
...
 
-TO-DO
+**Step 1: setup environment**
+
+If you are a first timer of CloudLab, we encourage you to read the CloudLab doc for an overview of how Artifact Evaluation is generally conducted on CloudLab.
+
+[CloudLab For Artifact Evaluation](https://docs.cloudlab.us/repeatable-research.html#%28part._aec-members%29)
+
+If you do not already have a CloudLab account, please apply for one following this [link](https://www.cloudlab.us/signup.php),
+  and ask the SOSP AEC chair to add you to the SOSP AEC project. Please let us know if you have trouble accessing cloudlab, we can help set up the experiment and give you access.
```
Comment (marshtompsxd) [inline AE.md:59]: @tylergu  Can you change SOSP to OSDI?

Comment (tylergu) [reply AE.md]: fixed

**Comment by marshtompsxd** [comment]
Ready to merge?

**Comment by tylergu** [comment]
Yes

**Pull request merged**